code/modules/mob/living/carbon/human/species/station/station_special_abilities_vr.dm

```diff
@@ -796,3 +796,138 @@
+			src.visible_message("<font color='red'>[src] sinks their fangs into [T]!</font>")
+			T.bloodstr.add_reagent("numbing_enzyme",5)
+		else if(choice == "Paralyzing")
+			src.show_message("<span class='warning'>You sink your fangs into [T] and inject your poison!</span>")
+			src.visible_message("<font color='red'>[src] sinks their fangs into [T]!</font>")
+			T.bloodstr.add_reagent("succubi_paralize",5)
+		else
+			return //Should never happen
+
+/*
+mob/living/carbon/proc/charmed() //TODO
+	charmed = 1
+
+	spawn(0)
+		for(var/i = 1,i > 0, i--)
+			src << "<font color='blue'><i>... [pick(charmed)] ...</i></font>"
+		charmed = 0
+
+*/
+
+/datum/reagent/succubi_aphrodisiac
+	name = "Aphrodisiac"
+	id = "succubi_aphrodisiac"
+	description = "A unknown liquid, it smells sweet"
+	color = "#8A0829"
+	scannable = 0
+
+/datum/reagent/succubi_aphrodisiac/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
+	if(prob(7))
+		M.show_message("<span class='warning'>You feel funny, and fall in love with the person in front of you</span>")
+		M.emote(pick("blush", "moans", "giggles", "turns visibly red"))
+		//M.charmed() //TODO
+		return
+/*
+/datum/reagent/succubi_numbing //Using numbing_enzyme instead.
+	name = "Numbing Fluid"
+	id = "succubi_numbing"
+	description = "A unknown liquid, it doesn't smell"
+	color = "#41029B"
+	scannable = 0
+
+/datum/reagent/succubi_numbing/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
+*/
+
+/datum/reagent/succubi_paralize
+	name = "Paralyzing Fluid"
+	id = "succubi_numbing"
+	description = "A unknown liquid, it doesn't smell"
+	color = "#41029B"
+	scannable = 0
+
+/datum/reagent/succubi_paralize/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
+	if(prob(7))
+		M.show_message("<span class='warning'>You lose sensation of your body.</span>")
+		M.Weaken(10)
+		return
+
+/mob/living/carbon/human/proc/face_sit()
+    set name = "Face Sit"
+    set desc = "Sit on your Preys Face"
+    set category = "Abilities"
+
+    if(last_special > world.time)
+        return
+
+    if(!ishuman(src))
+        return //If you're not a human you don't have permission to do this.
+
+    var/mob/living/carbon/human/C = src
+
+    var/obj/item/weapon/grab/G = src.get_active_hand()
+
+    if(!istype(G))
+        to_chat(C, "<span class='warning'>You must be grabbing a creature in your active hand to sit on them.</span>")
+        return
+
+    var/mob/living/carbon/human/T = G.affecting
+
+    if(!istype(T) || T.isSynthetic())
+        to_chat(src, "<span class='warning'>\The [T] is not able to be sit on.</span>")
+        return
+
+    if(G.state != GRAB_AGGRESSIVE)
+        to_chat(C, "<span class='warning'>You must have the creature pinned on the ground to sit on them </span>")
+        return
+
+    src.visible_message("<font color='red'><b>[src] moves their ass to [T]'s head, sitting down on them, making them unable to see anything else than [src]'s butt </b></font>")
```

Comment:
It seems as if sitting doesn't knock them down on the ground? So the target would still be standing up.